packages/rum/src/boot/startRecording.spec.ts

```diff
@@ -52,6 +52,9 @@ describe('startRecording', () => {
         },
       })
       .withSession(session)
+      .withConfiguration({
+        initialPrivacyLevel: InitialPrivacyLevel.ALLOW,
```

Comment:
With the default privacy level (mask forms only) data compression is too efficient (🎉), so segments are not reaching the compressed size limit.